mobile/src/table.js

```diff
@@ -205,9 +209,8 @@ export default class MyList extends Component {
     )
   }
 
-  renderHeaderButtons = (x, y, z) => {
-    const { isAdmin } = this.props
-    if (isAdmin)
+  renderHeaderButtons = (x, y, z, isSessionAdmin) => {
```

Comment:
x,y, and z don't seem to be used

Reply:
x, y, z are used starting on line 235

Reply:
ah, I missed them, sorry.  Maybe name them `popularStyle`, `recentStyle`, `answeredStyle`